README.md

```diff
@@ -344,6 +352,42 @@ options outside of this section are shared.
   launch time. As described below, a `ConfigMap` can be used to point the
   plugin at a desired configuration file when deploying via `helm`.
 
+**`DEVICE_FILTER_ENABLED`**:
+  enable device filter (select/exclude) from node's discovered devices
+
+  `[true | false] (default 'false')`
+
+  This flag allows enable device filter feature. When set to `false`, it does
+  nothing even if the belows flags `DEVICE_FILTER_SELECT_DEVICES`
+  and `DEVICE_FILTER_SELECT_DEVICES` are set.
```

Comment:
```suggestion
  nothing even if the belows flags `DEVICE_FILTER_SELECT_DEVICES`
  and `DEVICE_FILTER_EXCLUDE_DEVICES ` are set.
```

Reply:
Totally agree, this was my copy-paste mistake. Fixed!